src/main/java/stresso/trie/CompactLL.java

```diff
@@ -20,23 +20,24 @@ public static void main(String[] args) throws Exception {
 
     if (args.length != 3) {
       System.err
-          .println("Usage: " + Split.class.getSimpleName() + " <fluo props> <max> <cutoff>");
+          .println("Usage: " + Split.class.getSimpleName() + " <fluo conn props> <max> <cutoff>");
       System.exit(-1);
     }
 
     FluoConfiguration config = new FluoConfiguration(new File(args[0]));
 
+    try(FluoAdmin admin = FluoFactory.newAdmin(config)) {
+      // Get the application config stored in zookeeper which has Accumulo connection properties
+      config = new FluoConfiguration(config.orElse(admin.getApplicationConfig()));
```

Comment:
what is this doing?

Reply:
This class needs the info to connect to accumulo.  That info is not in the connections props file, so it gets it from zookeeper.

Reply:
Ok after looking at the code again this makes sense.